backend/src/v4/routes/maps.js

```diff
@@ -546,39 +494,26 @@ function requestMapTile(req, res, domain, uri) {
 	});
 }
 
-function requestHereMapTile(req, res, domain, uri) {
-	uri += "?apiKey=" + config.here.apiKey;
+function requestHereMapTile(req, res, domain, resource, style, features) {
+	const featureList = features ? features.split(",") : [];
 
-	if (req.query.congestion) {
-		uri += "&congestion=" + req.query.congestion;
-	}
-	if (req.query.lg) {
-		uri += "&lg=" + req.query.lg;
-	}
-	if (req.query.lg2) {
-		uri += "&lg2=" + req.query.lg2;
-	}
-	if (req.query.min_traffic_congestion) {
-		uri += "&min_traffic_congestion=" + req.query.min_traffic_congestion;
-	}
-	if (req.query.pois) {
-		uri += "&pois=" + req.query.pois;
-	}
-	if (req.query.ppi) {
-		uri += "&ppi=" + req.query.ppi;
-	}
-	if (req.query.pview) {
-		uri += "&pview=" + req.query.pview;
+	let uri = `/v3/${resource}/mc/${req.params.zoomLevel}/${req.params.gridx}/${req.params.gridy}/png8?apiKey=${config.here.apiKey}`;
+
+	if (req.query.features) {
+		featureList.concat(req.query.features.split(","));
 	}
-	if (req.query.range) {
-		uri += "&range=" + req.query.range;
+
+	if (featureList.length > 0) {
+		uri += `&features=${featureList.join()}`;
 	}
+
 	if (req.query.style) {
-		uri += "&style=" + req.query.style;
-	}
-	if (req.query.time) {
-		uri += "&time=" + req.query.time;
+		uri += `&style=${req.query.style}`;
```

Comment:
similarly here, a quick grep suggests our unity game doesn't ever append this, so don't bother with supporting it. 

Unless i'm wrong (it was a very quick grep!)

Reply:
We don't make use of this from Unity. It was in case we ever wanted to override the styles; e.g. day/night modes

Reply:
Removed